src/go-chaincode/download-fabric.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -x
+echo "######## Begin download Fabric ########"
+
+echo "######## Extracting and decompressing Fabric version ${HLF_VERSION} ########"
+curl -O -L https://github.com/hyperledger/fabric/archive/v${HLF_VERSION}.tar.gz
```

Comment:
Could this be replaced with `go get -u github.com/hyperledger/fabric/core/chaincode/shim` in the build script? (That seems to be how the tutorial does things http://hyperledger-fabric.readthedocs.io/en/latest/chaincode4ade.html )

Reply:
I'm not sure if this allows us to version the package. To be honest, I don't know how important it is to version the package but it seems like @jorgedr94 had this for a reason.